Client/mods/deathmatch/logic/luadefs/CLuaBrowserDefs.cpp

```diff
@@ -172,6 +172,9 @@ int CLuaBrowserDefs::RequestBrowserDomains(lua_State* luaVM)
         {
             for (auto& strURL : pages)
             {
+                if (!strURL.empty())
```

Comment:
"if the url isn't empty => skip it" ?
I think it should have been the opposite or did I miss something ?

Reply:
If the string is empty, the code shouldn't continue as it will result in a crash. There should be some sort of URL passed.

Reply:
The code does the opposite of what you wrote. Whenever **strURL** is _not empty_ it will continue with the next entry from **pages** and everytime **strURL** _is empty_ it will request the domain from the given URL.

Reply:
In a `for` loop, `continue` doesn't mean "continue to execute the code below", it means "stop here for that one and continue looping with the next entry".
Now that you cleary told us what logic you wanted for that part of the code, just remove the `!` and it will do what you said.

Reply:
I see how I may have confused both of you, but you guys already get the gist of it (at least that's how it seems). Also thanks for clearing up "continue" Citizen01. The most important thing is that the code fixes the bug though right?